AntennaTracker/tracking.cpp

```diff
@@ -31,9 +31,15 @@ void Tracker::update_vehicle_pos_estimate()
  */
 void Tracker::update_tracker_position()
 {
+    struct Location temp_loc = current_loc;
+    stationary = false;
+
     // update our position if we have at least a 2D fix
     // REVISIT: what if we lose lock during a mission and the antenna is moving?
-    if (!ahrs.get_position(current_loc) && (gps.status() >= AP_GPS::GPS_OK_FIX_2D)) {
+    if (!ahrs.get_position(current_loc)) {
```

Comment:
@magicrub i think it should pass now if you give it another go, failed due to some unrelated things. I think I have this if statement wrong its abit of a mind bender to work out, I will correct that and it should be good to go.